src/redux/ books /books.js

```diff
@@ -0,0 +1,49 @@
+const ADD_BOOK = 'bookstore/books/ADD_BOOK';
+const REMOVE_BOOK = 'bookstore/books/REMOVE_BOOK';
+
+const addBook = (id, title, author) => ({
+  type: ADD_BOOK,
+  id,
+  title,
+  author,
+});
+
+const removeBook = (id) => ({
+  type: REMOVE_BOOK,
+  id,
+});
+
+const initialState = [
+  {
```

Comment:
Even though your code is working well as expected but you have files and folders naming issues:

- [ ] Please remove the white spaces at the beginning of these folders `books` and `categories` and `configureStore.js` file and fix their importing lines in your `index.js` file and components because they cause compiling errors and when I fix them everything worked as expected.

![image](https://user-images.githubusercontent.com/68796232/202447190-80e65cb7-14f0-41b1-a45f-f4f9990e4b29.png)